src/panels/lovelace/components/hui-image.ts

```diff
@@ -66,11 +70,37 @@ export class HuiImage extends LitElement {
     if (this.cameraImage && this.cameraView !== "live") {
       this._startUpdateCameraInterval();
     }
+    this._intersectionObserver = new IntersectionObserver(
+      this.handleIntersectionCallback.bind(this),
+      {
+        threshold: [0.0],
+      }
+    );
+    this._intersectionObserver.observe(this);
   }
 
   public disconnectedCallback(): void {
     super.disconnectedCallback();
     this._stopUpdateCameraInterval();
+    if (this._intersectionObserver) {
+      this._intersectionObserver.disconnect();
+    }
+    this.imageVisible = undefined;
+  }
+
+  protected handleIntersectionCallback(entries) {
+    const wasImageVisible = this.imageVisible;
+    for (const entry of entries) {
+      this.imageVisible = entry.isIntersecting;
+    }
+    if (
+      this.imageVisible &&
+      !wasImageVisible &&
+      this._shouldStartCameraUpdates()
+    ) {
+      this._startUpdateCameraInterval();
+      this._updateCameraImageSrc();
+    }
```

Comment:
Since `imageVisible` is a `state` it will fire an update cycle with render, you can use `willUpdate` to check if `imageVisible` was changed and then perform an action based on the current value of `imageVisible`.

Reply:
fixed